test/js/node/missing-module.test.js

```diff
@@ -0,0 +1,7 @@
+import { expect, test } from "bun:test";
+
+test("not implemented yet module masquerades as undefined and throws an error", () => {
+  const missingModule = "node:missing" + "";
```

Comment:
surprising that transpiler does't implement this

Reply:
I did concat because in `import(missingModule)` without concat I get an error when running the script test